posts/elixir/1/index.md

```diff
@@ -617,19 +627,41 @@ iex(room1@COMPUTER-NAME)> Portal.Door.get(blue)
+
+You can assemble the release by running `mix release`. Let's also pass the `MIX_ENV=prod` environment variable, as we want to assemble a release optimized for production:
+
+    $ MIX_ENV=prod mix release
+    * assembling portal-0.1.0 on MIX_ENV=prod
+
+    Release created at _build/prod/rel/portal!
+
+    # To start your system
+    _build/prod/rel/portal/bin/portal start
+
+A release was created inside `_build/prod/rel/portal`. You can package everything in this directory into a zip file and ship it to production. Typically teams assemble their release at the end of their continuous integration pipeline. Feel free to explore this directory in your editor.
+
+Inside the `_build/prod/rel/portal` directory, you will find a `bin/portal` script, which is the entry-point to the system. From here you can start the system, run RPC commands, connect to a remotely running node and so on. Give it a try:
+
+    $ _build/prod/rel/portal/bin/portal start
+
+There is a lot more to releases and [the Elixir documentation](http://hexdocs.pm/mix/Mix.Tasks.Release.html) does a good job of outlining their ins and outs and all possible configurations.
+
 ## Wrapping up
 
-So we have reached the end of this guide on how to get started with Elixir! It was a fun ride and we quickly went from manually starting doors processes to shooting fault-tolerant doors for distributed portal transfers!
+So we have reached the end of this guide on how to get started with Elixir! It was a fun ride and we quickly went from manually starting doors processes to shooting fault-tolerant doors for distributed portal transfers, which we can package and ship to production.
 
 We challenge you to continue learning and exploring more of Elixir by taking your portal application to the next level:
 
   * Add a `Portal.push_left/1` function that transfers the data in the other direction. How can you avoid the code duplication existing between the `push_left/1` and `push_right/1` functions?
 
-  * Learn more about [ExUnit](http://elixir-lang.org/docs/stable/ex_unit/ExUnit.html), Elixir's testing framework, and write tests for the functionality we have built so far. Remember we already have a default structure laid out in the `test` directory.
+  * Learn more about [ExUnit](http://hexdocs.pm/ex_unit/ExUnit.html), Elixir's testing framework, and write tests for the functionality we have built so far. Remember we already have a default structure laid out in the `test` directory.
 
-  * Generate HTML documentation for your project with [ExDoc](http://github.com/elixir-lang/ex_doc).
+  * Generate HTML documentation for your project with [ExDoc](https://github.com/elixir-lang/ex_doc).
 
-  * Push your project to an external source, like [Github](https://github.com), and publish a package using the [Hex package manager](https://hex.pm).
+  * Push your project to an external source, like [GitHub](httpss://github.com), and publish a package using the [Hex package manager](https://hex.pm).
```

Comment:
`httpss` should be `https`

Reply:
Guess who has a messed up macbook keyboard?

Thanks for the catch!

Reply:
```suggestion
  * Push your project to an external source, like [GitHub](https://github.com), and publish a package using the [Hex package manager](https://hex.pm).
```